src/placesview.cpp

```diff
@@ -172,7 +172,7 @@ void PlacesView::onClicked(const QModelIndex& index) {
 
 void PlacesView::setCurrentPath(Fm::FilePath path) {
     currentPath_ = std::move(path);
-    if(path) {
+    if(currentPath_) {
         // TODO: search for item with the path in model_ and select it.
```

Comment:
> move c-ctor of FilePath doesn't free any resource

No move c-tor/assignment should ever free resources (as the resources are moved to other object).